### ISSUE TITLE
[404 page] - Recommended links are not accessible (gray 100 over the gray 100 background)

### ISSUE THREAD

**Issue by DragosRistici** [carbon-design-system/carbon-for-ibm-dotcom-website#970]
## 404 page recommended links are not accessible

The 404 page [is visible here](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/404/). The recommended links are Gray 100, but so is the container background. This raises an accessibility problem. 
<img width="1396" alt="Screenshot 2021-05-10 at 15 24 41" src="https://user-images.githubusercontent.com/48327647/117658807-d9651c80-b1a3-11eb-8fdd-f052b9453b6c.png">
<img width="618" alt="Screenshot 2021-05-10 at 15 25 07" src="https://user-images.githubusercontent.com/48327647/117658863-e84bcf00-b1a3-11eb-8f31-03e77316646e.png">

**Comment by ljcarot** [comment]
@DragosRistici @shixiedesign Let's redesign this using Carbon and address accessibility issues then.

**Comment by DragosRistici** [comment]
@ljcarot we're on it

**Comment by RobertaJHahn** [comment]
@shixiedesign @DragosRistici This issue does not include any filters that will bring it up in any of or routine meetings.  Do you want it included in a Design meeting agenda or assigned to a sprint?

**Comment by shixiedesign** [comment]
Hey taking this issue out of design and adding `dev` because it's a bug on Gatsby template! The sooner we get some attention the better. Might be a cherry pick?

**Comment by jeffchew** [comment]
This is part of the gatsby theme so should be moved to that project.
